treeherder/model/models.py

```diff
@@ -772,12 +754,22 @@ def who(self):
             return "autoclassifier"
 
     @classmethod
-    def create(cls, job_id, bug_id, user=None, bug_open=False):
+    def create(cls, *, job_id, internal_bug_id=None, bugzilla_id=None, user=None, bug_open=False):
```

Comment:
what is the `*` as a parameter?

Reply:
It is Python 3's signature syntax to mean the following arguments are keyword only arguments. It is done by default when setting a default value (for example `internal_bug_id=None`) but in this case it also enforces it for `job_id`.
That means it is not possible to call `BugJobMap.create('some_job_id')`, but instead it is required to call `BugJobMap.create(job_id='some_job_id')`. It can help with arguments management or refactoring.